### ISSUE TITLE
Not working on vscode 1.43.1

### ISSUE THREAD

**Issue by lp35** [amir9480/vscode-cpp-helper#1]
Hi,

First thank you for working on this missing feature of vscode.

I can't get this extension working though. Right clicking on the declaration and clicking "Create implementation" does not give any error nor result.

on a side not, it is also impossible for me to set 
```
"CppHelper.SourcePattern": [
    "{FILE}.cpp",
    "{FILE}.c",
    "{FILE}.inl",
    "/src/{FILE}.cpp"
]
```

in my settings: 

![image](https://user-images.githubusercontent.com/17080848/77420097-c0e04a00-6dc9-11ea-8136-42142eebf6bc.png)

Thanks for your help.

**Comment by lp35** [comment]
Hi,

First thank you for working on this missing feature of vscode.

I can't get this extension working though. Right clicking on the declaration and clicking "Create implementation" does not give any error nor result.

on a side not, it is also impossible for me to set 
```
"CppHelper.SourcePattern": [
    "{FILE}.cpp",
    "{FILE}.c",
    "{FILE}.inl",
    "/src/{FILE}.cpp"
]
```

in my settings: 

![image](https://user-images.githubusercontent.com/17080848/77420097-c0e04a00-6dc9-11ea-8136-42142eebf6bc.png)

Thanks for your help.

**Comment by amir9480** [comment]
Hello @lp35 
Thanks for using my extension and sorry for English in advance.

First, for the configuration, I checked and that is because of a mistake in `package.json`. I gonna fix that as soon as possible and you can ignore the warning for now.

And for the implementation, I need more information to debug.
Please send me your sample header code and tell me exactly where you right-clicked so that I can check that with my regexes that I created for extension.

**Comment by lp35** [comment]
```
#ifndef ABOUTDIALOG_HPP
#define ABOUTDIALOG_HPP

#include <QDialog>

namespace Ui {
class AboutDialog;
}

class AboutDialog : public QDialog
{
    Q_OBJECT

public:
    explicit AboutDialog(QWidget *parent = 0);
    virtual ~AboutDialog();
    void test();

private:
    Ui::AboutDialog *ui;
};

#endif // ABOUTDIALOG_HPP
```

I put my mouse on test function and it does not generate anything...

**Comment by amir9480** [comment]
@lp35 I tested your code on my vscode and this is the result.
![screenshot](https://user-images.githubusercontent.com/16279288/77532584-c6fe2580-6eb2-11ea-941e-060feb269c24.gif)

Look like the problem is something else.
Please do these and send me the result.

- Press F1 and command `Disable All Installed Extensions for this Workspace`.
- Then enable `C++ Helper` from extensions.
- Press F1 and command `Developers: Reload Window`
- After reload Go to `Help > Toggle Developer Tools` and click on the console tab.
- Right-click on your code and use `Create Implementation` or `Create Header Guard`.
- Right-click on any Error and click `Save as` and save the log file.
- Send the file here.

At last, you can enable your extensions by command `Extensions: Enable All Extensions`.

And also tell what OS you are using.
Thanks.

Screenshots:
![image](https://user-images.githubusercontent.com/16279288/77530782-7df8a200-6eaf-11ea-9212-a88f0b2af367.png)
![image](https://user-images.githubusercontent.com/16279288/77530853-95378f80-6eaf-11ea-8399-f94ab0f04390.png)
![image](https://user-images.githubusercontent.com/16279288/77530886-9ec0f780-6eaf-11ea-893e-26219a51bcd2.png)
![image](https://user-images.githubusercontent.com/16279288/77530923-b0a29a80-6eaf-11ea-938f-d8db0ec2b6eb.png)
![image](https://user-images.githubusercontent.com/16279288/77530942-bc8e5c80-6eaf-11ea-85cb-50ef8c739a4a.png)
![image](https://user-images.githubusercontent.com/16279288/77531179-227ae400-6eb0-11ea-976a-5b6e600ee196.png)
![image](https://user-images.githubusercontent.com/16279288/77531299-4f2efb80-6eb0-11ea-8dd6-4500a7483216.png)

**Comment by lp35** [comment]
Hey,

Thank you for pointing me on the developer console. I guess I've found the bug:

![image](https://user-images.githubusercontent.com/17080848/77538938-d3798280-6ea0-11ea-896d-957c24954b0f.png)

I think you are doing your tests under MS Windows, where the NTFS filesystem is case insensitive. Your extension is looking for **H**elpers.js, and in my extension folder under my home directory ~/.vscode/extensions/amiralizadeh9480.cpp-helper-0.0.4/out, it is **h**elpers.js

Renaming files with the correct case fixes the problem, and I'm now able to generate methods. You should then update your file to respect the case.

Thank you very much, and good luck against Covid19, I've seen that your country is particularly affected. We pray for you guys.

Regards

**Comment by amir9480** [comment]
Hi.

The issue solved in v0.0.5.
The problem was I changed `helpers.ts` to `Helpers.ts` before but compiled file name was still `helpers.js` (Thanks to windows filesystem!). So I cleared compiled files and recompiled my codes.
I tested on Ubuntu 18.04 and v0.0.5 works fine.
